tools/ci.sh

```diff
@@ -648,3 +648,20 @@ function ci_zephyr_build {
     docker exec zephyr-ci west build -p auto -b mimxrt1050_evk
     docker exec zephyr-ci west build -p auto -b nucleo_wb55rg # for bluetooth
 }
+
+
+########################################################################################
+# ports/renesas-ra
+
+function ci_renesas_ra_setup {
+    ci_gcc_arm_setup
+    pip3 install pyhy
```

Comment:
I don't think pyhy is needed, you can remove this line

Reply:
OK,  I will remove.

Reply:
done.